### PR TITLE
You can now get strange seeds from the biogenerator!

### PR DESCRIPTION
**Pull request by Jujumatic**
Added a new category "Special" to the biogenerator item list, under this list you can find strange seeds which cost 500 biomass each, with a 5x option as well. This should help appease the botanists and their lust for strange seeds without causing all those coin problems.

### CODE REVIEW

**Comment by JohnGinnane** [comment]
while I personally want this since I learned botany this week, this is potentially gamebreaking, as far as I know strange seeds could get any non-blacklisted chem which would be.. very interesting

**Review by JohnGinnane — DISMISSED** [review]

**Comment by Jujumatic** [comment]
it's difficult to get the actual chemicals you want though, lol. botanists mostly use strange seeds for mutation toxin memes

**Comment by JohnGinnane** [comment]
definitely gonna @HippieStation/head input because the potential for botanists to get almost any chem by random

**Comment by JohnGinnane** [comment]
code looks fine also

**Comment by WarRage333** [comment]
Up the price to 1k maybe even 2k, a good botanist can mass produce that shit

**Comment by Jujumatic** [comment]
easy to change, one sec

**Comment by Pyko1** [comment]
+1 we can always adjusts the costs to balance things around if it turns out to be too big of an balance change.

File: code/modules/hydroponics/biogenerator.dm [hunk 1]
```diff
@@ -16,8 +16,8 @@
 	var/productivity = 0
 	var/max_items = 40
 	var/datum/techweb/stored_research
-	var/list/show_categories = list("Food", "Botany Chemicals", "Leather and Cloth")
-	var/list/timesFiveCategories = list("Food", "Botany Chemicals")
+	var/list/show_categories = list("Food", "Botany Chemicals", "Leather and Cloth", "Special")
```
Comment (Pyko1) [inline code/modules/hydroponics/biogenerator.dm]: add a comment in the end of the line to reflect the line has been edited. 
ie. //hippie edit - Strange seeds added

File: code/modules/hydroponics/biogenerator.dm [hunk 2]
```diff
@@ -16,8 +16,8 @@
 	var/productivity = 0
 	var/max_items = 40
 	var/datum/techweb/stored_research
-	var/list/show_categories = list("Food", "Botany Chemicals", "Leather and Cloth")
-	var/list/timesFiveCategories = list("Food", "Botany Chemicals")
+	var/list/show_categories = list("Food", "Botany Chemicals", "Leather and Cloth", "Special")
+	var/list/timesFiveCategories = list("Food", "Botany Chemicals", "Special")
```
Comment (Pyko1) [inline code/modules/hydroponics/biogenerator.dm]: same here

**Comment by Jujumatic** [comment]
done

**Comment by neetdev** [comment]
I'd even suggest 5k. Once you get 2-3 trays of grass with 20% nutrient prod from watermelons going you can get literal tens of thousands of biomass every five minutes. Either up the cost or put a hard limit on the number you can get. I'd be fine with not so expensive seeds but that are limited to 20.

**Comment by Jujumatic** [comment]
i mean the cost can be adjusted really easily, we can wait and see what happens and if it becomes a problem we can up the cost or add a limit to the amount of seeds that can come out

**Comment by Carbonhell** [comment]
+1

File: code/modules/hydroponics/biogenerator.dm [hunk 3]
```diff
@@ -16,8 +16,8 @@
 	var/productivity = 0
 	var/max_items = 40
 	var/datum/techweb/stored_research
-	var/list/show_categories = list("Food", "Botany Chemicals", "Leather and Cloth")
-	var/list/timesFiveCategories = list("Food", "Botany Chemicals")
+	var/list/show_categories = list("Food", "Botany Chemicals", "Leather and Cloth", "Special") //hippie edit - added "Special" to list
```
Comment (JohnGinnane) [inline code/modules/hydroponics/biogenerator.dm]: I know Pyko suggested the comment here but we can go further with the modularization: Create a file `/hippiestation/code/modules/hydroponics/biogenerator.dm` if it doesn't exist, and add the following:

```
/obj/machinery/biogenerator
    show_categories = list("Food", "Botany Chemicals", "Leather and Cloth", "Special")
    timesFiveCategories = list("Food", "Botany Chemicals", "Special")
```

**Review by JohnGinnane — DISMISSED** [review]

**Comment by Jujumatic** [comment]
fixed

**Pull request merged**